tests/Microsoft.Identity.Test.Unit/BrokerTests/BrokerRequestTests.cs

```diff
@@ -280,6 +280,7 @@ public void GetAccountWithDuplicateBrokerAccountsTest()
             broker.IsBrokerInstalledAndInvokable().Returns(true);
```

Comment:
nit: Previously (https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/4359097e755ddfd9bb1181f6f9655e57d3cca823) `PlatformProxyFactoryTests.cs ` had an extra assert.